### ISSUE TITLE
Hard to visually distinguish application status (DFO)

### ISSUE THREAD

**Issue by gobyrne** [GCTC-NTGC/design-gc-digital-talent#4]
### Application status
Hard to see application status difference

![Image](https://github.com/GCTC-NTGC/design-gc-digital-talent/assets/2494551/3969b023-ca4b-4320-b9d8-884cb0930b18)

**Comment by gobyrne** [comment]
@Jerryescandon can you please take this into account in your ROD build. I don't think it's worth doing anything about before ROD though.

**Comment by gobyrne** [comment]
@Jerryescandon can you close this if you've addressed or have decided not to address?

**Comment by Jerryescandon** [comment]
Yeah this should be addressed when updating the "All candidates" table, since we no longer have a "Status" column, but we do now have a "Final decision" column that has colour coded pills.

Related issue: https://github.com/GCTC-NTGC/gc-digital-talent/issues/8675

![image](https://github.com/GCTC-NTGC/design-gc-digital-talent/assets/16977254/c32722f5-3039-4bf4-bf0d-f763718d35c8)
